packages/block-library/src/table/style.scss

```diff
@@ -18,6 +18,13 @@
 		padding: 0.5em;
 	}
 
+	// Match caption styles across front end and editor
+	figcaption {
+		@include caption-style-theme();
```

Comment:
It seems there is a convention for mixins with the suffix `-theme` to only use them in the theme styles — is it an issue to create an inconsistency here?

Reply:
I think so yes :)

Reply:
Thanks both, that makes sense. I've removed the use of this mixin here and instead used specific styles. I went with:

```
	figcaption {
		color: inherit;
		font-size: max(13px, 80%);
		text-align: center;
	}
```

The only difference here is that the font size is set to a minimum of 13px, as this is the same size that was set in the mixin, but then I've also allowed it to be 80% of the font size set by the theme. I'm not sure if this is too opinionated, but I thought it would allow for a more theme-related font size.